libs/wingsdk/src/cloud/api.ts

```diff
@@ -173,23 +173,13 @@ export type CorsHeaders = {
  * @inflight `@winglang/sdk.cloud.IApiClient`
  */
 
-export abstract class Api extends Resource {
-  /**
-   * Creates a new cloud.Api instance through the app.
-   * @internal
-   */
-  public static _newApi(
-    scope: Construct,
-    id: string,
-    props: ApiProps = {}
-  ): Api {
-    return App.of(scope).newAbstract(API_FQN, scope, id, props);
-  }
-
+export class Api extends Resource {
```

Comment:
Even though these are no longer abstract in TypeScript, I'm wondering if we should still track their "abstract"-ness in some way, e.g. with an `@abstract` flag in the docstring

Reply:
Added, both for classes and members that are "abstract"